src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,42 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CalculatorTest {
+
+    private Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+
+    @AfterEach
+    void tearDown() {
+    }
+
+    @Test
+    void add() {
+        assertEquals(7, calculator.add(3, 4));
+    }
+
+    @Test
+    void subtract() {
+        assertEquals(3, calculator.subtract(7,4));
+    }
+
+    @Test
+    void multiply() {
```

Comment:
unit test name should correspond what exactly you testing

A lot of naming convention are present, just use one

checkIfCalculatorDivideCorrectlyLargeNumbers
 
given_validNumbers_shouldCorrectlyDivide
 
shouldThrowExceptionWhenDivideByZero
shouldCorrectlyDivideLargeNumbers

Few samples.

I prefer last one shoud...
but you could use any convenient for you.